### DISCUSSION TITLE
Wagtail 5.0 Release Candidate 1

### DISCUSSION THREAD

**Discussion by gasman** [wagtail/wagtail#10372] [Announcements]
The first Release Candidate of Wagtail 5.0 is now available! Major new features in this release include SVG image support, object usage information when deleting objects, greater customisability of snippets to bring them to parity with ModelAdmin, dark mode, and much more.

For the full details, see the release notes: https://docs.wagtail.org/en/latest/releases/5.0.html

We'd like to invite all Wagtail users to try this release candidate on their development and staging sites and give us your feedback, to help us catch any issues ahead of the final 5.0 release (scheduled for the start of May). To install, set `wagtail==5.0rc1` in your requirements file. For general notes on upgrading, see https://docs.wagtail.org/en/latest/releases/upgrading.html

**Comment by pymike00** [comment]
Excited about this one! Thank you all!
